src/admin.html

```diff
@@ -23,7 +23,7 @@ <h1>OL Hotel Admin Access </h1>
                 <div id="tab__main">
                   <h2 class="h2--date">Today's Date: <span id="todays__date"></span></h2>
                   <h2 class="h2__rooms--available" id="label__rooms--available--js">Total Rooms Available Today: <span id="rooms__today">loading...</span></p>
-                  <h2 class="h2__revenue" id="label__revenue--js">Todays Revenue: $ <span class="span__loading">loading...</span></h2>
+                  <h2 class="h2__revenue" id="label__revenue--js">Todays Revenue: $ <span id="span__revenue">loading...</span></h2>
```

Comment:
Not sure if it's possible to use something other than a span, just for a more semantic element.

Reply:
I'll open an issue for that, thanks!